File: C++/2335. Minimum Amount of Time to Fill Cups.cpp
class Solution {
public:
    int fillCups(vector<int>& a) {
        sort(a.begin(),a.end());
        int c=0;
        while(a[1]!=0 && a[2]!=0){
            c++;
            a[1]--;
            a[2]--;
            sort(a.begin(),a.end());
        }
        return c+a.back();
    }
};

File: JAVA/2335. Minimum Amount of Time to Fill Cups.java
class Solution {
    public int fillCups(int[] a) {
        Arrays.sort(a);
        int c=0;
        while(a[1]!=0 && a[2]!=0){
            c++;
            a[1]--;
            a[2]--;
            Arrays.sort(a);
        }
        return c+a[2];
    }
}